.ci/test.sh

```diff
@@ -159,10 +159,12 @@ elif [[ $TASK == "bdist" ]]; then
         ARCH=$(uname -m)
         if [[ $ARCH == "x86_64" ]]; then
             PLATFORM="manylinux1_x86_64"
+            # Make sure we can do both CPU and GPU; see tests/python_package_test/test_dual.py
+            export LIGHTGBM_TEST_DUAL_CPU_GPU=1
         else
             PLATFORM="manylinux2014_$ARCH"
         fi
-        cd $BUILD_DIRECTORY/python-package && python setup.py bdist_wheel --plat-name=$PLATFORM --python-tag py3 || exit -1
+        cd $BUILD_DIRECTORY/python-package && python setup.py bdist_wheel --integrated-opencl --plat-name=$PLATFORM --python-tag py3 || exit -1
```

Comment:
Please update docs: https://github.com/microsoft/LightGBM/pull/3660/files#diff-e14b183376b1177323f6e7245d8ad64f2cd26f9638c129769d6b3c0ba4698dd5R26.

Reply:
Will do, thanks for the reminder!

Reply:
Done in 594a711

Reply:
Thanks a lot! But I think that `integrated-opencl` build option shouldn't be exposed to the public interface.
Could you please revert the following added sections?

![image](https://user-images.githubusercontent.com/25141164/174461389-26f14f2e-1823-438c-be18-aaf163b75269.png)

Reply:
Done!